packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -594,6 +594,9 @@ export function getDeviceSpec() {
     }
     return {};
 }
+
+export const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000;
```

Comment:
It would be great to have very bold comment above it explaining what it is, why we need to have that process, what it means to change it. Consider pointing maybe to some shared readme.md describing key concepts about GC and just adding key aspects of this variable. For example, if my understanding is correct, it can't change after this PR is merged. Maybe it can be only increased over time, but not decreased? Please tell next developer how they need to act not to screw up

Reply:
We need a readme describing the GC process anyway. That might be a couple of months out until the sweep implementation is completed. Maybe we should add a comment here explaining the relevant bits here and update to point to the readme once its available?

Reply:
I'd suggests starting it now, even if it's mostly empty, and start putting references from code and filling in sections based on these references. We will never be able to find these places to update after the fact.

Reply:
That sounds reasonable. I will start putting together a document for GC.